resolwe_bio/processes/workflows/amplicon.yml

```diff
@@ -3,7 +3,7 @@
   data_name: "{{ reads|sample_name|default('?') }}"
   requirements:
     expression-engine: jinja
-  version: 3.0.1
+  version: 3.1.0
```

Comment:
IMO, removing a process from workflow is equal to removing an output from the process, which is a breaking change. Raise version accordingly and prepend changelog entry with "backward-incompatible"...